fuels-core/Cargo.toml

```
@@ -24,5 +24,5 @@ thiserror = { version = "1.0.30", default-features = false }
 hex = { version = "0.4.3", default-features = false, features = ["std"] }
 
 [features]
-default = ["fuel-types/default", "sha2"]
-no-std = ["fuel-types/serde-types-minimal"]
+default = ["fuel-types/default", "sha2", "std"]
+std = []
```

You'll likely need to make more of the deps optional, and only enable their "std" features when enabled in this crate.

Haven't tested myself but you'll need to change it to something like this:

```suggestion
std = ["hex/std", "fuel-tx/std", "fuel-types/std", "thiserror"]
```

---

Oh, interesting. TIL. Thanks!